install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -814,6 +814,11 @@ data:
   enable-k8s-terminating-endpoint: {{ .Values.enableK8sTerminatingEndpoint | quote }}
 {{- end }}
 
+{{- if hasKey .Values "dnsPolicyUnloadOnShutdown" }}
+  # Unload DNS policy rules on graceful showdown
+  dns-policy-unload-on-shutdown: {{.Values.dnsPolicyUnloadOnShutdown | quote }}
```

Comment:
just curious on how we decide which flag should be handled explicitly here, .Values.extraConfig seems to be the catch-all for others.